example.py

```diff
@@ -0,0 +1,51 @@
+import yaml
+import parastell
+import numpy as np
+
+# Load configuration from YAML file
+config_file = 'config.yaml'
+with open(config_file, 'r') as file:
+    config = yaml.safe_load(file)
+
+# Access radial_build configurations
+radial_build = config['radial_build']
+phi_list = radial_build['phi_list']
+theta_list = radial_build['theta_list']
+repeat = radial_build['repeated'] 
+wall_s = radial_build['wall_s'] 
+
+# Correctly access stellarator_parameters
+stellarator_parameters = config['stellarator_parameters']
+num_phi = stellarator_parameters['num_phi']
+num_theta = stellarator_parameters['num_theta']
+
+# Prepare the build dictionary according to your requirements
+build = {
+    'phi_list': phi_list,
+    'theta_list': theta_list,
+    'wall_s': wall_s,
+    'radial_build': radial_build['components'],
+    'components': {}
+}
+
+# Convert thickness_matrix in 'radial_build' components to numpy arrays
+for component, details in radial_build['components'].items():
+    if 'thickness_matrix' in details and isinstance(details['thickness_matrix'], list):
```

Comment:
What if it's not a list? is in an error?

Reply:
Yeah that was my experience, but I'll run some more tests and check it out again.

Reply:
This may be a question for @connoramoreno , but one wanted a uniform thickness blanket, how would it be defined in the current struture? Would it still be a theta/phi grid with very few points (2x2)? Or could it be a scalar for each thickness?  Do we have to handle cases like that?

Reply:
I'll ask Conner about it before the Thursday meeting.

Reply:
Sorry, the notification for this got lost in my email inbox. In the current implementation, components of uniform thickness are still defined on a grid but all elements have equal value. We could choose to handle input for uniform thickness components differently, of course.

Reply:
We should probably issue an error if this data is NOT a list, then.